diff --git a/meta/recipes-kernel/linux/kernel-devsrc.bb b/meta/recipes-kernel/linux/kernel-devsrc.bb
index 8a900ed182..0c10051be3 100644
--- a/meta/recipes-kernel/linux/kernel-devsrc.bb
+++ b/meta/recipes-kernel/linux/kernel-devsrc.bb
@@ -162,7 +162,9 @@ do_install() {
             cp -a --parents arch/arm64/kernel/vdso/note.S $kerneldir/build/
             cp -a --parents arch/arm64/kernel/vdso/gen_vdso_offsets.sh 
$kerneldir/build/
 
-            cp -a --parents arch/arm64/kernel/module.lds $kerneldir/build/ 
2>/dev/null || :
+            if [ -f "arch/arm64/kernel/module.lds" ]; then
+               cp -a --parents arch/arm64/kernel/module.lds $kerneldir/build/ 
2>/dev/null || :
+            fi
        fi
 
        if [ "${ARCH}" = "powerpc" ]; then
@@ -186,7 +188,9 @@ do_install() {
                cp -a --parents $SYSCALL_TOOLS $kerneldir/build/
             fi
 
-            cp -a --parents arch/arm/kernel/module.lds $kerneldir/build/
+            if [ -f "arch/arm/kernel/module.lds" ]; then
+               cp -a --parents arch/arm/kernel/module.lds $kerneldir/build/
+            fi
        fi
 
        if [ -d arch/${ARCH}/include ]; then